charts/mlp/values.yaml

```diff
@@ -77,7 +77,16 @@ gitlab:
   clientId: #clientId
   clientSecret: #clientSecret
 
+# If you would like to use an external postgres database, you can connect using these credentials
+externalPostgres:
+  enabled: false
```

Comment:
Can we not use another toggle? and only use `prostgresql.enabled` as source of truth for this behavior.

Reply:
Okay I think this makes things less complicated. Have updated the PR.